### PR TITLE
Fix format section of vertex input data processing

### PR DESCRIPTION
**Pull request by sfricke-samsung**
As discussed in #81 

@abbaswasim let me know if this looks good for you, removed the redundant part you were aiming for (as I agree it was wrong and misleading before) and moved the part I wanted to a new example

### CODE REVIEW

File: chapters/vertex_input_data_processing.md [hunk 1]
```diff
@@ -255,6 +232,62 @@ const VkVertexInputAttributeDescription attributes[] = {
...
+    0,                          // binding
+    sizeof(Vertex),             // stride
+    VK_VERTEX_INPUT_RATE_VERTEX // inputRate
+};
+
+const VkVertexInputAttributeDescription attributes[] = {
+    {
+        0,                          // location
+        binding.binding,            // binding
+        VK_FORMAT_R32G32_SFLOAT,    // format - Reads in two 32-bit signed floats
```
Comment (abbaswasim) [inline chapters/vertex_input_data_processing.md:262]: If you want to be explicit about this, perhaps adding which floats will be read at the end so 

`// format - Reads in two 32-bit signed floats` becomes `// format - Reads in two 32-bit signed floats "a" and "b"`

File: chapters/vertex_input_data_processing.md [hunk 2]
```diff
@@ -255,6 +232,62 @@ const VkVertexInputAttributeDescription attributes[] = {
...
+        0                           // offset
+    },
+    {
+        1,                          // location
+        binding.binding,            // binding
+        VK_FORMAT_R32G32B32_SFLOAT, // format - Reads in three 32-bit signed floats
```
Comment (abbaswasim) [inline chapters/vertex_input_data_processing.md:268]: Same here `// format - Reads in three 32-bit signed floats` becomes `// format - Points at three 32-bit signed floats "b", "c" and "d"` providing a hint that the last component will be discarded later.

**Review by abbaswasim — APPROVED** [review]
Looks good thanks for making the changes. Just added two minor comments.

File: chapters/vertex_input_data_processing.md (hunk 1 above)
Comment (sfricke-samsung) [reply chapters/vertex_input_data_processing.md:262]:
```suggestion
        VK_FORMAT_R32G32_SFLOAT,    // format - Reads in two 32-bit signed floats ('a' and 'b')
```

File: chapters/vertex_input_data_processing.md (hunk 2 above)
Comment (sfricke-samsung) [reply chapters/vertex_input_data_processing.md:268]:
```suggestion
        VK_FORMAT_R32G32B32_SFLOAT, // format - Reads in three 32-bit signed floats ('b', 'c', and 'd')
```

**Pull request merged**